### DISCUSSION TITLE
Feature request: Search and preview Obsidia callouts (`oc`)

### DISCUSSION THREAD

**Discussion by flavinsh** [chrisgrieser/shimmering-obsidian#182] [Ideas]
This 👆.

I frequently use the `ocss` keyword to open `callout.css` where I manage custom [Obsidian Callouts](https://help.obsidian.md/Editing+and+formatting/Callouts). Searching and previewing them from Alfred would be the icing on the cake (e.g. using the `oc` keyword).

**Comment by chrisgrieser** [comment]
Previewing callouts is not sth that can be done properly via Alfred, so this type of feature would have to be implemented via an Obsidian plugin, sorry.
